apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/BorrowDetailsGraph.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC, useState } from 'react';
+
+import { t } from 'i18next';
+
+import { Accordion, Link, Paragraph } from '@sovryn/ui';
+
+import { AmountRenderer } from '../../../../2_molecules/AmountRenderer/AmountRenderer';
+import { StatisticsCard } from '../../../../2_molecules/StatisticsCard/StatisticsCard';
+import { useIsMobile } from '../../../../../hooks/useIsMobile';
+import { translations } from '../../../../../locales/i18n';
```

Comment:
If posible, absolute paths could be defined at tsconfig to avoid import hell!

Reply:
Totally agree, but project doesn't have it already and we don't want to introduce such a project level change  )